packages/integration-karma/test/light-dom/basic/index.spec.js

```diff
@@ -0,0 +1,24 @@
+import { createElement, setFeatureFlagForTest } from 'lwc';
+
+import Test from 'x/test';
+describe('Basic Light DOM', () => {
+    it('should render properly', () => {
+        setFeatureFlagForTest('ENABLE_LIGHT_DOM_COMPONENTS', true);
```

Comment:
It isn't safe to flip flags in the middle of a test. If the test fails, the flag value is never reset to its original value. I would recommend moving this to a `before` and an `after` block.

Reply:
This observation is still valid. The flag enablement and disablement should be extracted outside the test in a `before` and `after` block.